package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "jhipster-registry",
-  "version": "7.0.0",
+  "version": "7.2.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "version": "7.0.0",
+      "version": "7.2.0",
```

Comment:
```suggestion
      "version": "7.1.0",
```

Reply:
should be unchanged (so 7.0.0), as this version will be updated when I'll do the release